components/switch/README.md

```diff
@@ -71,13 +71,27 @@ You can also add labels for your checked and unchecked values by using component
 </gameface-switch>
 ```
 
+## Switch attributes
+
 You can use the following attributes to customize the switch
 
 | Attribute           | Type    | Default | Description                                | Notes                                                                                                                                                                                                                 |
 | ------------------- | ------- | ------- | ------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| type              | String  | default       | The type of the switch       | It can be default, inset or text-inside                                                                                                                                                                       |
```

Comment:
There is a default "state" which is different from `inset`:
![image](https://github.com/CoherentLabs/GameUIComponents/assets/21058220/5681e956-c6ff-4a26-a7f3-2e506f5d40eb)
We can remove the default value to be empty.

Reply:
Removed